inspirehep/modules/records/indexer.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import, division, print_function
+
+from invenio_indexer.api import RecordIndexer
+
+from inspirehep.modules.records.api import InspireRecord
+
+
+class InspireRecordIndexer(RecordIndexer):
+    def index_by_id(self, record_uuid):
+        """Index a record by record identifier.
+        :param record_uuid: Record identifier.
```

Comment:
Currently we are using Napoleon style for docstrings :point_right:  https://sphinxcontrib-napoleon.readthedocs.io/en/latest/